Editor/ScaleAdjuster/ScaleAdjusterPreview.cs

```diff
@@ -322,7 +322,7 @@ public void OnFrame(Renderer original, Renderer proxy)
 
             var rootBone = _finalBonesMap.TryGetValue(smr.rootBone, out var newRootBone) ? newRootBone : smr.rootBone;
             smr.rootBone = rootBone;
-            smr.bones = smr.bones.Select(b => _finalBonesMap.GetValueOrDefault(b, b)).ToArray();
+            smr.bones = smr.bones.Select(b => b is null ? null : _finalBonesMap.GetValueOrDefault(b, b)).ToArray();
```

Comment:
Should this use `== null` to check for Unity destroyed-on-native-side objects as well?